src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughFallThroughLotsOfComment.java

```diff
@@ -0,0 +1,61 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughFallThroughLotsOfComment {
+
+    void method(int lineNumber) {
+        switch (lineNumber) {
+            case 3:
+                Object pi = null;
+                int line = 2;
+                // fall-through intended - the line contains already the first entries
+                // the following 4 lines contain additional satellite ids
+            case 4:
+            case 5:
+            case 6:
+            case 7: {
+                int lineLength =  2;
+                int startIdx = 9;
+                while (lineLength++ < 2&& (startIdx + 3) <= lineLength) {
+                    String satId;
+                    startIdx += 3;
+                }
+                break;
+            }
+
+            case 14:
+                // ignore additional custom fields
+                break;
+
+            // position/velocity/clock components
+            case 15: {
+                // String base = line.substring(3, 13).trim();
+
+                // base = line.substring(14, 26).trim();
+            }
+            case 16: // violation 'Fall through from previous branch of the switch statement'
```

Comment:
Why is this a violation? Are we counting empty block as code?

Reply:
> may be yes, because current implementation is showing violation

```
kevin@inspiron-15-5510:~/Desktop/check_style$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="FallThrough">
        </module>
    </module>
</module>

kevin@inspiron-15-5510:~/Desktop/check_style$ cat Test.java
public class Test
 {
     void method(int i, int j, boolean cond) {
         while (true) {
             switch (i) {
             case 0: 
             case 1:{

                 }
             case 2: 
                 i++;
                 break;
            }
        }
    }
 }
kevin@inspiron-15-5510:~/Desktop/check_style$ java -jar /home/kevin/Downloads/checkstyle-10.10.0-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /home/kevin/Desktop/check_style/Test.java:10:14: Fall through from previous branch of the switch statement. [FallThrough]
Audit done.
Checkstyle ends with 1 errors.

```